interface/main/tabs/templates/patient_data_template.php

```diff
@@ -41,8 +39,8 @@
                     <?php echo xlt("None");?>
                 <!-- /ko -->
                 <!-- ko if: patient -->
-                    <a class="css_button_small" href="#" class="clear" data-bind="click:clearPatient" title="<?php echo xlt("Clear") ?>">
-                        <i style="font-size:150%;" class="fa fa-times"></i>
+                    <a class="btn btn-xs btn-link" href="#" class="clear" data-bind="click:clearPatient" title="<?php echo xlt("Clear") ?>">
```

Comment:
I am not sure how I missed this when this was brought in, but this should be xla() since within html attribute.